docs/user/dashboard/tutorials.asciidoc

```diff
@@ -382,19 +371,11 @@ To fix this, try updating the `encoding` of your Vega-Lite spec to:
   }
 ```
 
-This will show more errors, and you can inspect `VEGA_DEBUG.view.data('data_0')` to
+This will show more errors, and you can inspect `data_0` to
```

Comment:
```suggestion
This will show more errors, so you need to debug. Click *Inspect*, switch the view to *Vega Debug*, and switch to look at the visually encoded data in `data_0` to
```